hack/install-controller-gen.sh

```diff
@@ -6,6 +6,7 @@ CONTROLLER_TOOLS_VERSION=${1:-v0.4.0}
 OUTPUT=${2:-${ORIG_DIR}/bin/controller-gen}
 
 WORK_DIR=$(mktemp --tmpdir -d controller-gen-XXX)
+cp -r config/render ${WORK_DIR}/
```

Comment:
This should not be in this file. we should nto touch instal-controller-gen.sh what needs to be modified is generate.sh, to include the new file too : https://github.com/metal3-io/baremetal-operator/blob/master/hack/generate.sh#L18